include/dxc/dxcapi.h

```diff
@@ -291,6 +291,7 @@ static const UINT32 DxcVersionInfoFlags_Internal = 2; // Internal Validator (non
 struct __declspec(uuid("b04f5b50-2059-4f12-a8ff-a1e0cde1cc7e"))
 IDxcVersionInfo : public IUnknown {
   virtual HRESULT STDMETHODCALLTYPE GetVersion(_Out_ UINT32 *pMajor, _Out_ UINT32 *pMinor) = 0;
+  virtual HRESULT STDMETHODCALLTYPE GetCommitInfo(_Out_ UINT32 *pCommitCount, _Out_ const char **pCommitHash) = 0;
```

Comment:
This is changing a COM interface, which breaks users & implementers. The pattern to use is to declare an IDxcVersionInfo2 interface derived from IDxcVersionInfo, and QI for that as needed (with the expectation that some clients might implement only the basic version, not the extended one).

Reply:
SG. I'll create a `IDxcVersionInfo2` for this new method. I think we also need a new UUID for it?

Besides, do you think it's useful to also add a parameter for the commit branch?

Reply:
Done. I've grabbed the UUID using `uuidgen` by following https://msdn.microsoft.com/de-de/library/windows/desktop/aa373928(v=vs.85).aspx. It seems that the UUID is not tight to the class name in any sense. Is this expected? Am I doing it correctly?

Reply:
Correct UUIDs aren't tied to names in general (there's fine print around WinRT template/generic interfaces, but none of that applies here). Thanks!